qiskit/converters/circuit_to_dag.py

```diff
@@ -16,7 +16,7 @@
 from qiskit.dagcircuit.dagcircuit import DAGCircuit
 
 
-def circuit_to_dag(circuit, copy_operations=True):
+def circuit_to_dag(circuit, copy_operations=True, *, qubit_order=None, clbit_order=None):
```

Comment:
As it is, this function could be used to create a DAG that doesn't have the same qubits as `circuit`, at least in the case where the bits aren't in a register or used by an instruction, right? Perhaps we should instead take iterables of indices to reorder by, or do some checking to make sure all bits from `circuit` are accounted for, and no extra bits were provided.

Reply:
I'm not a huge fan of providing indices into a foreign list because it tends to be annoying to compute and it can be confusing whether it's a forwards or backwards permutation ("put bit `i` in new position `order[i]`" or "put bit `order[i]` in new position `i`" - both make sense), and we'd still need to check for duplicate indices and the correct length of list anyway.  I can put in a check that the bits are an exact permutation, though, no problem.

Reply:
Done in 7b4e79179.